lock/src/main/java/com/auth0/android/lock/LockActivity.java

```diff
@@ -87,15 +91,21 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             return;
         }
 
-        lockBus = new Bus();
+        Bus lockBus = new Bus();
         lockBus.register(this);
         handler = new Handler(getMainLooper());
 
         setContentView(R.layout.com_auth0_lock_activity_lock);
-        progress = (LockProgress) findViewById(R.id.com_auth0_lock_progress);
-        rootView = (LinearLayout) findViewById(R.id.com_auth0_lock_content);
+        resultMessage = (TextView) findViewById(R.id.com_auth0_lock_result_message);
+        RelativeLayout rootView = (RelativeLayout) findViewById(R.id.com_auth0_lock_content);
+        panelHolder = new ClassicPanelHolder(this, lockBus);
+        rootView.addView(panelHolder, ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.MATCH_PARENT);
+
+        fetchApplication();
```

Comment:
In order to be consistent we should use the event to trigger the fetch (FetchApplicationEvent)